src/RoleObjects/AccessibilityObject.ts

```diff
@@ -15,15 +65,19 @@ export default class AccessibilityObject {
      * Fields with relatively fixed values that should go into the React props for the
      * element translation of this object.  This is done as an object for easy merging
      * with the rest of the props
+     *
+     * ts-ignore for dispatchEvent with 3 args
+     *
      * @access package
      */
     this._reactProps = {
       key: this._displayObject.id,
       id: this.domId,
-      onFocus: (evt) => {
+      onFocus: (evt: Event) => {
         const cancelled = this._displayObject.dispatchEvent(
           'focus',
           false,
+          // @ts-ignore
```

Comment:
- there is a mismatch in the number of args in `@types/createjs` (https://github.com/DefinitelyTyped/DefinitelyTyped) so we had to override here
- we intend on opening a PR to resolve